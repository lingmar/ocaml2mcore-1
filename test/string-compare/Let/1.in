let a = -3 in
let b = a in
let c = b in
c
