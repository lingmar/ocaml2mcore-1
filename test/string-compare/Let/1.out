mexpr
let a = negi 3 in
let b = a in
let c = b in
c
